apps/live-search/public/index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Live Search</title>
+    <meta
+      name="description"
+      content="A utility API proxy for the freeCodeCamp curriculum"
+    />
+    <link
+      rel="icon"
+      type="image/png"
+      href="https://cdn.freecodecamp.org/universal/favicons/favicon.ico"
+    />
+    <link rel="stylesheet" href="/styles.css" />
+  </head>
+  <body>
+    <main>
+      <h1>Live Search API</h1>
+      <div class="usage">
+        <h2>Usage</h2>
+        <p>
+          Use the endpoint
+          <code>https://live-search.freecodecamp.rocks/api/?q={query}</code>
+          to filter through fruits, where <code>query</code> is your query
+          parameter.
+        </p>
+        <h2>Example Requests</h2>
+        <ul>
+          <li>
+            <a href="/api/pokemon" target="_blank" rel="noreferrer noopener"
+              ><code>https://live-search.freecodecamp.rocks/api/?q=a</code></a
+            >
+            <span
+              ><b>:</b> returns a list of fruits containing the letter
+              <code>a</code></span
+            >
+          </li>
+          <li>
+            <a href="/api/pokemon" target="_blank" rel="noreferrer noopener"
+              ><code
+                >https://live-search.freecodecamp.rocks/api/?q=berry</code
+              ></a
+            >
+            <span
+              ><b>:</b> returns a list of fruits containing the letters
```

Comment:
it looks like the `b` element is only be used for styling purposes. So I would suggest changing it to a `span` and then using css to make it bold.